projects/libs/core-library-typescript/test/virtual-machine/virtual-machine.spec.ts

```diff
@@ -50,6 +54,23 @@ describe("test virtual machine", () => {
     virtualMachine = new VirtualMachine(stateStorage);
   });
 
+  it("rejects a transaction with a non-positive amount", async () => {
+    const transaction = aTransaction({ from: "account1", to: "account2", amount: 0 });
+
+    const { processedTransactions, stateDiff, rejectedTransactions } = await virtualMachine.processTransactionSet({
+      orderedTransactions: [transaction]
+    });
+
+    rejectedTransactions.should.have.lengthOf(1);
```

Comment:
Can you change to `expect`? We're trying to use a single idiom in our tests (alas, without too much luck).

Reply:
Well, the test was like this before and I wasn't sure if this was on purpose or not. I'll change.